package.json

```diff
@@ -74,7 +74,7 @@
     "ethers": "^5.5.3",
     "long": "^5.2.0",
     "siwe": "^2.1.3",
-    "xmtpv3_wasm": "github:xmtp/libxmtp#vodozemac_dev"
+    "xmtpv3_wasm": "github:xmtp/libxmtp#michaelx_export_and_import_of_vodozemac_identities"
```

Comment:
Will change the branch once the other PR: https://github.com/xmtp/libxmtp/pull/40 goes in